BungeeCord-Patches/0064-Adventure.patch

```diff
@@ -284,11 +289,11 @@ index fc92a7b5..6eda2d70 100644
 +    }
 +
 +    // Waterfall start - we're going on an adventure!
-+    public net.kyori.adventure.text.Component description() {
++    public net.kyori.adventure.text.@org.checkerframework.checker.nullness.qual.NonNull Component description() {
```

Comment:
here also same search&replace mistake, not going to comment at all places but there are more

Reply:
Same as above